connectors/hub-servicenow/src/main/service/rhel/files/default-conf/application.properties

```diff
@@ -0,0 +1,11 @@
+# vIDM public key url (you must override this in /etc)
```

Comment:
woops :)

Reply:
Somehow I have missed this. We can change as part of APF-1687

Reply:
Yea I missed it too: https://github.com/vmware/connectors-workspace-one/blob/APF-1539_hub_concur_connector/connectors/hub-concur/src/main/service/rhel/files/default-conf/application.properties